internal/nice/nice.go

```diff
@@ -12,6 +12,11 @@ type Nice struct {
 	NonInteractive bool
 }
 
+// NewNice returns a new Nice instance
+func NewNice() *Nice {
+	return &Nice{}
+}
+
```

Comment:
Not needed as stated in other comments
```suggestion

```

Reply:
as mentioned above, the problem here is this `NewNice()` lack of initialised `NonInteractive` field.